src/Server/MiddlewareAliasRegistry.php

```diff
@@ -67,7 +69,15 @@ function ($middleware) {
 
     private function aliasExists(string $alias): bool
     {
-        return isset($this->registry[$alias]);
+        [$aliasName,] = $this->parseAlias($alias);
+
+        return isset($this->registry[$aliasName]);
     }
 
-}
\ No newline at end of file
+    private function parseAlias(string $alias): array
+    {
+        $parts = mb_split(':', $alias, 2);
```

Comment:
Why do we need mb_split here (and require mbstring)? I see no way for plain ASCII split to fail when splitting an UTF8 string by ":".

Reply:
Ok, got it. It could be caused by my habit to use `mb_` everywhere by default :)

Will change.